server/dive_server/viame.py

```diff
@@ -266,18 +260,11 @@ def run_training(self, folderIds, pipelineName, config):
             if folder is None:
                 raise RestException(f"Cannot access folder {folderId}")
             folder_names.append(folder['name'])
-            detections = list(
-                Item().find({"meta.detection": str(folderId)}).sort([("created", -1)])
-            )
-            detection = detections[0] if detections else None
-
-            if not detection:
-                raise RestException(f"No detections for folder {folder['name']}")
-
             # Ensure detection has a csv format
             # TODO: Move this into worker job
-            csv_detection_file(folder, detection, user)
-            detection_list.append(detection)
+            detections_json_item = detections_item(folder, strict=True)
+            ensure_csv_detections_file(folder, detections_json_item, user)
+            detection_list.append(detections_json_item)
```

Comment:
Little confusing at first glance because  we are using a  json named ITEM to store both the json and the csv.  This `detection_json_item` could already have an attached csv.  I don't believe we assigned detection metadata to any csv's anymore so all detection_items at least have a json file?  Maybe favor using `detection_item` here?

Reply:
I did this just because the name of the function was so similar to the variable that it was a readability issue.  Changed to `train_on_detections_item`, is that OK?

Reply:
Yeah I completely forgot that would be a similar name to the function.